RELEASE.md

```diff
@@ -17,6 +17,32 @@ Please follow the established format:
 
 <!-- Add release notes for the upcoming release here -->
 
+# Release 3.8.0
+
+## Major features and improvements
+
+- Finish the new node metadata side panel, which allows you to switch between different modular pipelines. This feature is no longer hidden behind a flag, and the 'meta' flag has been removed. (#293, #309, #312, #320, #321, #326, #295, #329, #333)
+- Enable the new graphing layout algorithm by default, and remove the 'newgraph' feature flag. If necessary, you can still revert back to the old layout algorithm by enabling the 'oldgraph' flag. (#334, #335)
+- Lazy-load sidebar node-list rows on scroll. This will improve performance by not forcing the app to render the entire node-list on larger graphs. This feature is disabled by default, but can be enabled using the 'lazy' feature flag. (#307)
+- Support launching a development server against a real pipeline (#318, #327)
```

Comment:
I don't think we should mention this as a feature since this isn't tried and tested within our team yet - we are still investigating the error that were returned from running the dev server using the new bash command, it is definitely worth more testing, and if successful, more detailed documentation on the readme / contribution docs before we announce this as part of the major feature.

Reply:
I could move it to the minor feature list or mention that it's still a work-in-progress, but I think it should be mentioned, because as a general rule we always mention things that are included in a release, even works-in-progress and unfinished things, because I see the release notes partly as a first port of call for someone trying to debug a breakage.

Reply:
Agreed with Susanna on this.

Reply:
sure thing, moving it to minor feature would be good - it might also be worth to mention that it is WIP?

Reply:
@yetudada I still think we should mention it (for the above reasons), so would it be okay to just edit it to mention that it's a WIP?

Reply:
Done